snap/snapcraft.yaml

```diff
@@ -71,21 +72,41 @@ parts:
       - libqrencode-dev
     stage-packages:
       - ca-certificates
-      - libboost-chrono1.58.0
-      - libboost-test1.58.0
-      - libboost-thread1.58.0
-      - libevent-2.0-5
-      - libevent-core-2.0-5
-      - libevent-pthreads-2.0-5
+      - libboost-chrono1.65-dev
+      - libboost-test1.65-dev
+      - libboost-thread1.65-dev
+      - libboost-filesystem1.65.1
+      - libevent-2.1-6
+      - libevent-core-2.1-6
+      - libevent-pthreads-2.1-6
       - libminiupnpc10
-      - libprotobuf9v5
+      - libprotobuf-c-dev
+      - libprotobuf10
       - libqrencode3
       - libqt5test5
-      - libsodium18
+      - libsodium-dev
       - libzmq5
     after:
       - berkeleydb
-      - desktop-qt5
+      - qt
+  qt:
```

Comment:
I think it'd be useful to have a comment suggesting why this is needed. IIUC this just provides the `desktop-launch` command used above. You could take the comment from https://github.com/ryanofsky/packaging/commit/c49d5df81beea63fd505aad934bce043060374ad from when I tried to get this working previously.

Reply:
I have changed the name and added your comment

Reply:
unrelated naming update: Could also update the `parts.bitcoin` name to say `parts.bitcoin-core` to be more specific

Reply:
Yes, good point.
I have changed the name of the snap and the part to bitcoin-core